client/landing/stepper/declarative-flow/internals/index.tsx

```diff
@@ -159,7 +159,7 @@ export const FlowRenderer: React.FC< { flow: Flow } > = ( { flow } ) => {
 	}, [ location ] );
 
 	useEffect( () => {
-		if ( isFlowStart() ) {
+		if ( flow.isSignupFlow && isFlowStart() ) {
```

Comment:
I think that depends on which conceptual layer these events reside. In this case, we can ask whether we are tracking things about whether a flow is implemented on Stepper or a general signup flow experience. In this case I think we are tracking for the latter, so I think it'd be better to abstract the name away from having anything Stepper specific. If we really want to track implementation specific info, I'd recommend that we use a prop instead of a name.

Does that make sense? :)